zero/src/main/java/mutiny/zero/Tube.java

```diff
@@ -0,0 +1,23 @@
+package mutiny.zero;
+
+import java.util.function.LongConsumer;
+
+public interface Tube<T> {
```

Comment:
Funny word :-)

Reply:
I looked at plumbing terms and thought this sounded better than sink, etc.